publisher/internal/analytics/ethereum/analytics.go

```diff
@@ -64,9 +70,12 @@ func init() {
 	transferSig = convertToEventSignature(transferEventHeader)
 	burnSig = convertToEventSignature(burnEventHeader)
 	collectSig = convertToEventSignature(collectEventHeader)
+	swapSig = convertToEventSignature(swapEventHeader)
 
 	stableCoins = []string{strings.ToLower(addressUSDT), strings.ToLower(addressUSDC)}
 	nativeCoins = []string{strings.ToLower(addressWETH)}
+
+	unknownAddressOccurrences = make(map[string]int)
```

Comment:
Please do not use global variables even for statistics.

One possible workaround would be to some-what overutilize singleton pattern and create a dedicated package that allows to gather and retrieve such statistics... But an even better approach would be to use e.g. Prometheus. This will open up to much bigger instrumentation possibilities.

A simpler approach would be to move this variable to respective `Operation` and then aggregate inside Analytics.